docs/data/material/components/toggle-button/ColorToggleButton.tsx

```diff
@@ -18,6 +18,7 @@ export default function ColorToggleButton() {
       value={alignment}
       exclusive
       onChange={handleChange}
+      aria-label="Color"
```

Comment:
Should we be more descriptive here?

Reply:
![image](https://user-images.githubusercontent.com/11643426/183073687-8eb6b6b3-830e-4d07-b368-f479fa15cfd1.png)

Yes sure, This is what it will speak for the screen readers users.
Can you suggest me something which I can replace with ?

Thanks 😃

Reply:
How about we use `Platform` here? The toggle buttons refer to different platforms, not colors :)

Reply:
Nice Suggestion.

Done.